Resources/public/js/views/services/ez-languageselectionboxviewservice.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-languageselectionboxviewservice', function (Y) {
+    "use strict";
+    /**
+     * Provides the language selection box view service class
+     *
+     * @method ez-languageselectionboxviewservice
+     */
+
+    /**
+     * The language selection box view service
+     *
+     * @namespace eZ
+     * @class LanguageSelectionBoxViewService
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.LanguageSelectionBoxViewService = Y.Base.create('languageSelectionBoxViewService', Y.eZ.ViewService, [], {
+        getViewParameters: function () {
+            var config = this.get('config');
+
+            config.availableTranslations = this.get('availableTranslations');
```

Comment:
by doing that you are modifying an object which is not *yours*, you should first *clone* (with `Y.merge`) config.